hub/apps/get-started/uno-simple-photo-viewer.md

```diff
@@ -0,0 +1,622 @@
+---
+title: Tutorial--Make a simple photo viewer that targets multiple platforms
+description: Learn how to reach users on other platforms like Web, iOS, Android, and Linux with minimal changes to the C#/WinUI 3 simple photo viewer built in the previous tutorial. We'll use Uno Platform to create a new multi-platform app, which we can move code from the existing desktop project to.
+ms.topic: article
+ms.date: 05/21/2023
+keywords: Windows, App, SDK, WinUI 3, WinUI, photo, viewer, Windows 11, Windows 10, XAML, C#, uno platform, uno
+ms.author: aashcraft
+author: alvinashcraft
+ms.localizationpriority: medium
+---
+
+# Tutorial: Make a simple photo viewer that targets multiple platforms
+
+After you've [created](/hub/apps/get-started/simple-photo-viewer-winui3.md) a starter simple photo viewer WinUI 3 app, you might be wondering how to reach more users without having to rewrite your app. This tutorial will use [Uno Platform](https://platform.uno/) to expand the reach of your existing C# WinUI 3 application enabling reuse of the business logic and UI layer across native mobile, web, and desktop. With only minimal changes to the simple photo viewer app, we'll be able to run a pixel-perfect copy of the app ported to platforms like the Web, iOS, Android, macOS, and Linux.
+
+[Insert picture here]
+
+## Prerequisites
+
+- [Visual Studio 2022 17.4 or later](https://visualstudio.microsoft.com/#vs-section)
+- [Tools for Windows App SDK](../windows-app-sdk/set-up-your-development-environment.md)
+- ASP.NET and web development workload (for WebAssembly development)
+:::image type="content" source="../images/uno/uno-vs-install-web.png" alt-text="Web development workload in VS":::
+- .NET Multi-platform App UI development installed (for iOS, Android, Mac Catalyst development).
+:::image type="content" source="../images/uno/uno-vs-install-dotnet-mobile.png" alt-text="dotnet mobile workload in VS":::
+- .NET desktop development installed (for Gtk, Wpf, and Linux Framebuffer development)
+:::image type="content" source="../how-tos/images/hello-world/vs-install-dotnet.png" alt-text=".net desktop workload in VS":::
+
+## Finalize your environment
+
+1. Open a command-line prompt, Windows Terminal if you have it installed, or else Command Prompt or Windows Powershell from the Start Menu.
```

Comment:
@alvinashcraft are we able to extract the "finalize your environment" steps (which are basically the steps for ensuring uno is setup) into a separate .md file that can be referenced both here and in [this doc](https://learn.microsoft.com/en-us/windows/apps/how-tos/uno-multiplatform) so that we can maintain these instructions in a single place?
Perhaps this should include the above pre-requisites too?

Reply:
Hey Nick. Yes, you can use include files. Here's the guidance on them: https://learn.microsoft.com/en-us/contribute/markdown-reference#includes-syntax

Let me know if you have questions about using them. You can put the "Includes" folder in one spot and use a relative link to reference it from both articles. You should be able to search the repo for other uses to get an idea of how it's used in practice.

Reply:
@lukeblevins can you give this a shot to share content between the two uno blog posts